src/components/Text/Text.tsx

```diff
@@ -21,13 +22,20 @@ export type Variant =
 export type Props = {
   /**
    * Controls whether to render text inline (defaults to "p");
+   * Use "div" to indicate usage of `<Text>` as a text passage,
+   * (i.e. wraps <p>, <h1>-<h6>, <a>, <ol>, <ul>, <blockquote>, <hr>)
    */
-  as?: 'p' | 'span';
+  as?: 'p' | 'span' | 'div';
+  /**
+   * Flags if the length of the text passage should be capped.
+   * Used only with as="div" and defaults to true
+   */
+  capLinelength?: boolean;
   children: React.ReactNode;
   className?: string;
   variant?: Variant;
   size?: Size;
-  spacing?: 'half' | '1x' | '2x';
+  bottomSpacing?: 'half' | '1x' | '2x';
```

Comment:
@anniehu4 how do you feel about this change? Discussion's here: https://github.com/chanzuckerberg/edu-design-system/pull/1083#discussion_r881572437

Reply:
thanks for the 👀 ! I'm good either way, just added some consideration to the thread